test/types/records/sungeun/emptyRecords/extern/externRecordInvalidEQUALS.good

```diff
@@ -1,2 +1,2 @@
 externRecordInvalidEQUALS.chpl:5: error: unresolved call '==(blah_t, 1)'
-externRecordInvalidEQUALS.chpl:1: note: this candidate did not match: ==(_arg1: blah_t, _arg2: blah_t)
+externRecordInvalidEQUALS.chpl:1: note: candidates are: ==(_arg1: blah_t, _arg2: blah_t)
```

Comment:
It doesn't seem like this change is related to your PR?  Is there any chance that you were running on the wrong side of a recent Michael PR that reworded this type of resolution error message, or with an outdated compiler?

Reply:
(or did the change in PREDIFF cause this?  I'm having trouble determining from the PR or commit messages what caused these and why it relates to string changes).

Reply:
This is actually related to this PR, but it's more random.  Because I added some arithmetic operators, some unresolved calls try to find candidates in a different order than before.

Reply:
I have been wondering if we want some tests to simply not print out the candidates. (I don't even think there's a flag for that today).

Reply:
It occurs to me that for these tests, I can just change the prediff to look at only the first line, which won't change when people add new operators that change the order in which candidates are found.  I'll do that.

Whether there's a larger effort to make tests in general not print out the candidates, I'll leave up to someone to decide with a different PR.